apis/goharbor.io/v1alpha2/harborcluster_types.go

```diff
@@ -113,11 +114,18 @@ type StorageRedirectSpec struct {
 	// Default is false
 	// +optional
 	Disable bool `json:"disable"`
```

Comment:
`Disable ` => `Enable`(default value=true)

```yaml
redirect:
  enable: true
  route: # optional
     tls: tls_secret # optional
     host: my.minio.io # required
```

Reply:
suggest using `expose` instead of `route`, because there is a `HarborExposeComponentSpec` in the types.

``` yaml
redirect:
   enable: true
   expose:
      tls: tls_secret
      ingress:
          host: my.minio.io
          controller: xxx
          annotations: xxx
```

Reply:
可以 @soulseen

Reply:
so we use `HarborExposeComponentSpec` directly, right? 

```yaml
Expose *HarborExposeIngressSpec `json:"expose,omitempty"`
```

Reply:
```yaml
redirect:
   enable: true
   expose:
      tls: 
        certificateRef: tls_secret
      ingress:
          host: my.minio.io
          controller: xxx
          annotations: xxx
```